### ISSUE TITLE
Cannot edit email signature in Outlook 2010 with NVDA

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#6471]
The issue looks like a duplicate of #3833 but seems unresolved.  Using NVDA 2016.3 on Windows 7.  The signature cannot be edited, even by someone using the mouse, with NVDA running.  Worked fine when switching to Window-Eyes.

The last line of the log file is:
AttributeError: 'NoneType' object has no attribute 'range'

Andrew

**Comment by jcsteh** [comment]
Probably a duplicate of #5890. However, you mentioned that "The signature cannot be edited, even by someone using the mouse". Can you clarify what you mean regarding the mouse? While I understand NVDA can't read this field, this shouldn't affect a sighted user typing into and reading the field.

**Comment by andrew-l-d** [comment]
Yes, the "someone using the mouse" was a sighted colleague. Clicking in 
the field did not allow editing.  I was most bemused.



Andrew

**Comment by leonardder** [comment]
@andrew-l-d: What is the state of this issue?
Given your findings that editing after clicking in the field with the mouse is not even possible, this looks like the signature is read only. Therefore I'm inclined to close this issue, but if you could provide another update first, that'd be nice.

**Comment by fernando-jose-silva** [comment]
[assinaturaoutlook.txt](https://github.com/nvaccess/nvda/files/1740630/assinaturaoutlook.txt)
I tried to edit an email signature using office 2013 by navigating with the keyboard and I could not, when on arriving with shift + tab in the signature edit field nvda makes an error, and when typing and trying to read with the arrows the text or nvda is muted.
You could see that typed text was entered by pressing tab and then chift + tab to have nvda read the text contained in the signature edit box.
I've attached a nvda log.

**Comment by andrew-l-d** [comment]
I cannot give an update re Outlook 2010, as we have moved to Outlook 2016.  It seems possible to create/edit signatures in Outlook 2016 with NVDA 2017.4.  It therefore seems appropriate to close.

Andrew

**Comment by leonardder** [comment]
It seems to be appropriate to close this indeed.

fernando-jose-silva: regarding your issue with Outlook 2013, please see https://github.com/nvaccess/nvda/issues/5890#issuecomment-366999514
